OptimizelySDKCore/OptimizelySDKCore/OPTLYProjectConfig.m

```diff
@@ -108,11 +110,20 @@ - (instancetype)initWithBuilder:(OPTLYProjectConfigBuilder *)builder {
         return nil;
     }
     
+    supportedDatafileVersions = @[@"2", @"3", @"4"];
```

Comment:
let's move this up to the top of the init method.   do you think we should wrap with a dispatch_once?  also, maybe a comment might be nice :)